### PR TITLE
CROS103:Check for get past events from steped

### PR DESCRIPTION
**Pull request by akayi07**

### CODE REVIEW

File: src/client.ts [hunk 1]
```diff
@@ -50,6 +50,9 @@ export class DelphinusContract {
   }
 
   async getPastEventsFromSteped(fromBlock: number, toBlock: number, step: number) {
+    if(fromBlock < toBlock) {
```
Comment (yymone) [inline src/client.ts]: ????? fromBlock < toBlock is correct scenario?

Comment (akayi07) [reply src/client.ts]: Yes, it's a mistake.

File: src/client.ts [hunk 2]
```diff
@@ -50,6 +50,9 @@ export class DelphinusContract {
   }
 
   async getPastEventsFromSteped(fromBlock: number, toBlock: number, step: number) {
+    if(fromBlock > toBlock) {
+      throw("Error: fromBlock > toBlock");
```
Comment (yymone) [inline src/client.ts]: Please show the number of  fromBlock and toBlock so it will be convenient for investigation when error happen.

Comment (akayi07) [reply src/client.ts]: Ok， I also updated screenshots in jira.

**Comment by yymone** [comment]
As currently our monitor input  usages may be "from = last latest + 1", "to = latest" (last latest maybe equal latest if latest block is slow and not increased) so we do not do this performance improvement currently.

**Pull request closed without merging**